test-suite/mock-spine-mhs-outbound/build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
-    id 'org.springframework.boot' version '2.7.1'
-    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
-    id 'io.freefair.lombok' version '6.2.0'
+    id 'org.springframework.boot' version '3.3.3'
```

Comment:
Wouldn't it be better if we allow the patch level to be the latest, leaving the major and minor levels as-is? E.g. `3.3.x`?

Reply:
I just took it from other project services where we are using minor version which currently is ver. 3.3.3,

Reply:
OK, no biggie - just thought I would highlight it 😄